lib/evm_database_ops.rb

```diff
@@ -67,11 +66,6 @@ def self.dump(db_opts, connect_opts = {})
     # db_opts and connect_opts similar to .backup
 
     uri = with_file_storage(:dump, db_opts, connect_opts) do |database_opts|
-      # For database dumps, this isn't going to be as accurate (since the dump
-      # size will probably be larger than the calculated BD size), but it still
```

Comment:
The advantage of moving this comment as well is it allowed me to cover up the "BD" typo that I made when I originally wrote this... 😅